tests/gtest/avifrgbtoyuvtest.cc

```diff
@@ -764,19 +766,21 @@ TEST_P(YUVToRGBThreadingTest, TestIdentical) {
   yuv->matrixCoefficients = AVIF_MATRIX_COEFFICIENTS_BT601;
   yuv->yuvRange = AVIF_RANGE_FULL;
 
-  // Fill YUV planes with random values.
+  // Fill YUVA planes with random values.
   srand(0xAABBCCDD);
   const int yuv_max = (1 << yuv_depth);
-  ASSERT_EQ(avifImageAllocatePlanes(yuv.get(), AVIF_PLANES_YUV),
+  ASSERT_EQ(avifImageAllocatePlanes(
+                yuv.get(), has_alpha ? AVIF_PLANES_ALL : AVIF_PLANES_YUV),
             AVIF_RESULT_OK);
-  avifPixelFormatInfo pixel_format_info;
-  avifGetPixelFormatInfo(yuv_format, &pixel_format_info);
-  const int plane_count = pixel_format_info.monochrome ? 1 : 3;
-  for (int plane = 0; plane < plane_count; ++plane) {
+  for (int plane = AVIF_CHAN_Y;
+       plane <= (has_alpha ? AVIF_CHAN_A : AVIF_CHAN_V); ++plane) {
```

Comment:
We can blindly go up to `AVIF_CHAN_A` here. If `yuv` doesn't have the alpha plane allocated, `avifImagePlaneWidth(yuv.get(), AVIF_CHAN_A)` returns 0 and line 778 will handle this case.